apps/api/apps/chat/views.py

```
@@ -45,7 +50,7 @@ def message(self, request):
             answer, references = rag_service.generate_answer(request.user, user_message)
         except Exception as e:
             # Fallback or generic error handling
-            answer = "Maaf, terjadi kesalahan saat menghubungi AI atau mencari dokumen."
+            answer = f"Maaf, terjadi kesalahan teknis saat menghubungi AI: {str(e)}"
```

Ruff (0.16.2)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)